helpers/constants.js

```diff
@@ -110,7 +110,10 @@ var constants = {
 		offset: 1451520, // Start rewards at block (n)
 		distance: 3000000, // Distance between each milestone
 	},
-	// WARNING: When changing totalAmount you also need to change getBlockRewards(int) SQL function!
+	// Block height at which block version increases
+	blockVersions: [
+		0,
+	],
```

Comment:
This is confusing to me. If I follow the comment above, it says: 

> at block height 0 the block version will be increased 

I believe better would be to manage an object with **starting block height** and associated **block version**

```
blockVersions: {
   1: 0,
   6521962: 1
},
```

This is just a thought, target is to make this constant more readable. May be some other way is also possible.

Reply:
Added comments and changed initial height from 0 to 1.